configurations/walmart/es6-node.js

```diff
@@ -4,5 +4,58 @@ module.exports = {
   "extends": [
     "defaults/configurations/walmart/es6",
     "defaults/rules/eslint/node/walmart"
-  ]
+  ],
+  "env": {
+    "es6": false
```

Comment:
We've got `defaults/configurations/walmart/es6` then this `false`. Should we just remove both the extends and the env?